ezBreezy.js

```diff
@@ -0,0 +1,10 @@
+//code for index - week 4 exercise
+
+function doubleLuck() {
+    let input=prompt('enter your lucky number');
+    let doubleNumber=input;
+    let result=0;
+
+    result=doubleNumber * Number(input);
+    alert(result);
+}
```

Comment:
Dont forget to put spaces around those assignment operators. Also the doubleNumber variable number isn't needed, plus you don't run Number() on it, which can lead to bugs.

`result = Number(input) * Number(input)`

```
function doubleLuck() {
    let input = prompt('enter your lucky number');
    let doubleNumber = input;
    let result = 0;

    result = doubleNumber * Number(input);
    alert(result);
}
```

Reply:
I see that I needed to put a space between the equal sign and the rest of the text, sorry, the assignment operators and the rest of the text (codepen lets you get away with it - bad, codepen!). Also I see what you mean about doubleNumber being redundant when I can just use a 2. But I am confused about the last note. The video makes it clear that the "Number" part of "Number(input)" is actually necessary - the instructor makes a point of saying so (he even changes his code from just "input" to "Number(input)"), because he tells us, the audience that he wants JavaScript to understand that he wants the input to be a number and not a string and therefore wrapping "input" with "Number" makes everything clear to JS. So, is that incorrect?

Reply:
Yes, you need to run Number() on strings if you want to use them as numbers. I see how you might have read what I wrote as saying DON'T run it ever, when I'm just pointing out that you don't run it on that string in your code. Maybe not my greatest sentence ever, but if you ever take a string from a user, and are expecting a number you will need the Number().